File: tests/expected/comment_unsupported.nim
proc do_unsupported() =
  let a = 1
  # dict comprehension ((key + 1), (value + 1)) unimplemented on line 9:4
  let b = bool(a)
  echo if b: "True" else: "False"

proc main() =
  do_unsupported()

main()

File: pyjl/tests_py2many-py2many/comment_unsupported.jl
function do_unsupported()
a = 1
Dict((key + 1)=>(value + 1) for ((key + 1), (value + 1)) in Dict());
b = Bool(a)
println(b ? ("True") : ("False"));
end

function main()
do_unsupported();
end

main()
